elide-integration-tests/src/test/java/com/yahoo/elide/tests/AnyPolymorphismIT.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright 2017, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.tests;
+
+import com.jayway.restassured.RestAssured;
+import com.yahoo.elide.core.HttpStatus;
+import com.yahoo.elide.initialization.AbstractApiResourceInitializer;
+import com.yahoo.elide.utils.JsonParser;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+
+public class AnyPolymorphismIT extends AbstractApiResourceInitializer {
+    private static final String JSONAPI_CONTENT_TYPE = "application/vnd.api+json";
+    private final JsonParser jsonParser = new JsonParser();
+
+    private final String sep = "/";
+
+    private final int horsepower = 102;
+    private final String one = "1";
```

Comment:
I realize codacy probably made you do this, but I personally find the tests harder to read with this indirection.

Reply:
This applies for all of these string constants.

Reply:
I 100% agree, is there a way to get codacy to like it?   Or do we ignore these in codacy?

Reply:
Codacy should now ignore files in `*/src/test/*`